src/utils.ts

```diff
@@ -249,6 +249,17 @@ export const updateParam = (param: { [x: string]: any }) => {
         s.int_padding = s.int_padding ?? (param.loop ? 250 : 355);
     });
 
+    // Version 5.4.13
+    if (param.branchSpacingPct === undefined) {
+        if (param.style === RmgStyle.SHMetro) {
+            param.branchSpacingPct = Math.round((param.branch_spacing / param.svg_height) * 300);
```

Comment:
300 might be a reasonable value in svg, but for older templates, converting with 200 will make the visual appearance the same as before.

Reply:
See https://github.com/railmapgen/rmg/blob/be8b86d8d33c69d1b9c45b7f9243c84b012e1b9b/src/svgs/railmap/main/main-shmetro.tsx#L69

Reply:
I have no objection in 300. The idea is that if update the old templates with 300, they are not the same as before in branches. Updating with 200 will solve the problem and looks the same. Also I can update the templates if 200 is merged.

Reply:
The difference here has nothing to do with the PR, it may be introduced in the previous commit. Let me update the templates.